### PR TITLE
Refresh product flags on ajax call when product page is refreshed

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Refresh product flags on ajax call when product page is refreshed
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/14321
| How to test?  | See ticket. Please clear your browser cache as this PR modifies the FO Javascript.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14325)

### CODE REVIEW

File: themes/classic/templates/catalog/_partials/miniatures/product.tpl [hunk 1]
```diff
@@ -82,6 +82,7 @@
         {/block}
       </div>
 
+      <!-- @todo: use include file='catalog/_partials/product-flags.tpl'} -->
```
Comment (matks) [inline themes/classic/templates/catalog/_partials/miniatures/product.tpl:85]: In order to reduce the number of changes so close to RC, I prefer to perform the refactoring in next version

File: themes/classic/templates/catalog/product.tpl [hunk 2]
```diff
@@ -57,6 +57,7 @@
         {block name='page_content_container'}
           <section class="page-content" id="content">
             {block name='page_content'}
+              <!-- @todo: use include file='catalog/_partials/product-flags.tpl'} -->
```
Comment (matks) [inline themes/classic/templates/catalog/product.tpl:60]: In order to reduce the number of changes so close to RC, I prefer to perform the refactoring in next version

File: themes/classic/templates/catalog/_partials/miniatures/product.tpl (hunk 1 above)
Comment (mickaelandrieu) [reply themes/classic/templates/catalog/_partials/miniatures/product.tpl:85]: fair enough

**Review by mickaelandrieu — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @matks,

In my case => it is OK
https://drive.google.com/file/d/1bOrsoCgfWx_srVwrlRNgsvZo83e33a--/view
Ping @sarahdib

Thanks!

**Comment by matks** [comment]
Thank you @mickaelandrieu for review and @sarahdib for QA

**Comment by Mreco83** [comment]
Hello. How do I fix an error in prestashop 1.7.5.2? In prestashop 1.7.5.2 I don't see a file like themes / _core / js / product.js

**Pull request merged**